### PR TITLE
spark language added in checkbox

### PR DESCRIPTION
**Pull request by ShouryaSrivastava01**
@chilarai please merge my PR. Thank you

### CODE REVIEW

File: A-Z-checkboxes.html [hunk 1]
```diff
@@ -1553,6 +1553,9 @@ <h1>S</h1>
 <input type="checkbox" id="script.net" name="pchbx" value="script.net" />
 <label for="script.net"> Script.NET </label><br />
 
+<input type="checkbox" id="spark" name="pchbx" value="spark" />
+<label for="script.net"> Spark </label><br />
```
Comment (chilarai) [inline A-Z-checkboxes.html:1557]: Please change the label for="script.net" and it should be merged

**Review by chilarai — CHANGES_REQUESTED** [review]
Thanks. But do the required changes before the merge

**Comment by ShouryaSrivastava01** [comment]
@chilarai changes applied

**Review by chilarai — APPROVED** [review]
Thank you

**Pull request merged**